es-rest-client-1.3/core/src/main/java/com/bazaarvoice/elasticsearch/client/core/AbstractClient.java

```diff
@@ -356,8 +358,10 @@
 
     @Override public ThreadPool threadPool() {
         // TODO flesh out client
-        final ThreadPool dummyThreadPool = new ThreadPool("dummy");
-        return dummyThreadPool;
+        if(this.threadPool == null) {
```

Comment:
I did not think this threadpool would be used for every request. If it is only referenced and not used, I'll make it static. If it is used, I'll check to make sure the one we're creating is appropriate.